### PR TITLE
Cleanup stuff

### PR DESCRIPTION
**Pull request by pkehrer**
Addresses:

* got rid of all my function() definitions
* moved swagger back to json, dropped yamljs dependency (i got into this by switching between swagger frameworks a couple times... I generally prefer json to yaml, not sure how you feel)
* api resource is now /tournament (https://github.com/daward/lovebot/issues/12)
* "standings seem off" (https://github.com/daward/lovebot/issues/6)
  * I think this is fixed. You already committed the change to actually show the number of games played, rather than doing gamesPerMatch * numberOfMatches.  I also had misnamed the parameter in the swagger docs as "matchesPerGame" which doesn't make sense... I reran your test case and got:
```
{
  "gameParameters": {
    "numberOfMatches": 100,
    "gamesPerMatch": 4,
    "totalGames": 688
  },
  "standings": [
    {
      "playerId": "Goodguess over HTTP",
      "matchWinPct": 0.48,
      "matchesWon": 48,
      "place": 1
    },
    {
      "playerId": "No pricness strat",
      "matchWinPct": 0.46,
      "matchesWon": 46,
      "place": 2
    },
    {
      "playerId": "Random",
      "matchWinPct": 0.06,
      "matchesWon": 6,
      "place": 3
    }
  ]
}
```

### CODE REVIEW

File: api/controllers/tournament.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
   body.gamesPerMatch = body.gamesPerMatch || bodyParam.schema.schema.properties.gamesPerMatch.default;
 
-  runner.runGame(body).then(response => res.json(response));
-}
\ No newline at end of file
+  runner(body).then(response => res.json(response));
+}
+
+module.exports = {
+  runTournament: runTournament
```
Comment (pkehrer) [inline api/controllers/tournament.js:14]: This still needs to export a function by name, because it is picked up by the swagger router.

Comment (daward) [reply api/controllers/tournament.js:14]: Ah, yes.  That makes sense.

File: api/api.js [hunk 2]
```diff
@@ -1,7 +1,6 @@
 let path = require('path');
 let fs = require('fs');
 let _ = require('lodash');
-let YAML = require('yamljs');
```
Comment (daward) [inline api/api.js:4]: I slightly prefer yaml, but it annoys me at times with the spacing.  So yeah, I dont care, if you want to get rid of dependency, lets do it.

File: readme.md [hunk 3]
```diff
@@ -1,11 +1,23 @@
...
 * `$ node index.js `
 
 ### runner API
 * `$ npm start`
-* Access swagger UI at http://localhost:3333/
\ No newline at end of file
+* Access swagger UI at http://localhost:3333/
+
+## deployment
+The app is currently deployed on heroku. In order to deploy, you must be added as a contributor on the heroku app by Paul.
```
Comment (daward) [inline readme.md:14]: Sweet.  Am I a contrib?

Comment (pkehrer) [reply readme.md:14]: you have a heroku account? I just sent an invite to acheron0@gmail.com...

**Pull request merged**